README.md

```diff
@@ -20,4 +37,32 @@ any values in the project's config.py.
 4. To run the development server, execute the following:
 ```bash
 % env/bin/python run.py
-```
\ No newline at end of file
+```
+
+## Static assets development
+
+Javascript and SASS assets are built against Node.js v8.9.3. Usage of
```

Comment:
Can we add the install step to dev_install.sh?

Reply:
Yes, I was considering that but was concerned that there's more than one way to install node.js and I wasn't sure if it would be more confusing than helpful to put in there. Would we be using dev_install.sh for build and test automation? Or just as a developer helper?

Reply:
It's just a developer helper